diagnostic.md

```diff
@@ -6,70 +6,89 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+
+
+get '/books/:id', to: 'books#show'
+
+Or
+
+get '/books/:id' => 'books#show'
+
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+in the routes.rb file
+
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+Bundle exec rake is a bundler command that will excecute the script db:migrate and when db:migrate migrates the data in the database.
+
 ```
 
 What is the difference between `rails g scaffold things` and
 `rails g controller things`?
 
 ```md
-// your response here
+the scaffold command generates routes, a controller and model for things. rails g controller generates just makes the  controller for things.
+
 ```
 
 What is the job of the `serializer`?
 
 ```md
-// your response here
+
+
 ```
 
 What do we expect the command `Patient.all` to return?
 
 ```md
-// your response here
+all the patient in the db
+
 ```
 
 We use the constant `Patient` in the `PatientsController`, but where is it
 actually defined?
 
 ```md
-// your response here
+
+in the model
+
 ```
 
 Why do our `model` files appear to be lacking so much code, when they're doing
 so much for us?
 
 ```md
-// your response here
+
+
 ```
 
 What does typing `bin/rails dbconsole` in a directory that houses a rails app do for
```

Comment:
This is a typo and should have been `bin/rails console` so no points will be deducted for this.